bin/trait_mapping/create_efo_table.py

```diff
@@ -32,6 +35,19 @@ def get_parent_terms(url):
     return [term['label'] for term in requests.get(url).json()['_embedded']['terms']]
 
 
+def uri_to_curie(uri):
+    """Converts URI to curie (short identifier)"""
```

Comment:
I recently found out about CURIEs so I don't need clarification of this, but could you add an example of the conversion for those not familiar with them?

Reply:
Fair point, added documentation on URIs and CURIEs.